From 756bee93c95fce2c1456932ff7fe1f727d773856 Mon Sep 17 00:00:00 2001
From: vcrhonek <vcrhonek@fedoraproject.org>
Date: Wed, 27 Aug 2014 13:39:53 +0200
Subject: [PATCH 04/14] Fix '\' can not be used to quote all delimiters

Related: #435421

Adjusted for tcsh-6.19.00 by Fridolin Pokorny <fpokorny@redhat.com>

---
 tcsh.man | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tcsh.man b/tcsh.man
index 4fa59d2..b79bafe 100644
--- a/tcsh.man
+++ b/tcsh.man
@@ -1047,7 +1047,7 @@ Substitute \fIl\fR for \fIr\fR.
 \fIl\fR is simply a string like \fIr\fR, not a regular expression as in
 the eponymous \fIed\fR(1) command.
 Any character may be used as the delimiter in place of `/';
-a `\e' can be used to quote the delimiter inside \fIl\fR and \fIr\fR.
+a `\\' can be used to quote the delimiter expect `(', `)', `|' and `>' inside \fIl\fR and \fIr\fR.
 The character `&' in the \fIr\fR is replaced by \fIl\fR; `\e' also quotes `&'.
 If \fIl\fR is empty (``''), the \fIl\fR from a previous substitution or the
 \fIs\fR from a previous search or event number in event specification is used.
-- 
1.9.3

